### ISSUE TITLE
[BUG] [Formatter] Incorrect indentation of custom template tags with a variable inside said template tag

### ISSUE THREAD

**Issue by malo-cayocca** [Riverside-Healthcare/djLint#572]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Ubuntu 20.04
 - Python Version 3.8.10
 - djLint Version 1.19.16
 - template language: django

## Issue
I have a custom template tag `flatblock` (declared in the `custom_blocks` of my `pyproject.toml`), which isn't correctly indented when only a variable is inside it, whereas the `block` template tag is correctly indented in the same situation :

#### Before formatting

```
{% block title %}{{ title }}{% endblock title %}

{% flatblock title %}{{ title }}{% endflatblock title %}
```

#### After formatting

```
{% block title %}
    {{ title }}
{% endblock title %}

{% flatblock title %}
    {{ title }}{% endflatblock title %}
```

Note that if I use an html tag in the `flatblock` it will be correctly indented : 

```
{% flatblock title %}
    <p>{{ title }}</p>
{% endflatblock title %}
```

## How To Reproduce

Here is the content of my `pyproject.toml`

```
[tool.djlint]
require_pragma = true
profile = "django"
custom_blocks = "stage, cache, flatblock"
blank_line_after_tag = "extends, endblock, endflatblock"
max_line_length = 120
ignore = "H021,H023,H030,H031"
```

**Comment by christopherpickering** [comment]
@malo-cayocca sorry for the huge delay in looking into to this. I should have a release out in a bit.

as a temporary workaround you can add "endflatblock" to your list of custom blocks:

```yml
custom_blocks = "stage, cache, flatblock, endflatblock"
```
